pro/src/data_access/ApiRecipeDataAccessObject.java

```diff
@@ -1,68 +1,83 @@
 package data_access;
 
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
 import okhttp3.OkHttpClient;
 import okhttp3.Request;
 import okhttp3.Response;
 import org.json.JSONObject;
+import service.search.use_case.SearchDataAccessInterface;
 
-import java.io.IOException;
-import java.util.HashMap;
-
-
-public class ApiRecipeDataAccessObject {
+public class ApiRecipeDataAccessObject implements SearchDataAccessInterface {
     private final String keyword;
 
-    public ApiRecipeDataAccessObject(String keyword){
+    public ApiRecipeDataAccessObject(String keyword) {
         this.keyword = keyword;
     }
 
-    public JSONObject content(String keyword) throws IOException {
+    private JSONObject content(String keyword) throws IOException {
```

Comment:
The method should catch the IOException because checked exceptions for conditions from which the caller can reasonably be expected to recover, in other words, we should give proper information when the exception occurs.